subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -364,74 +364,64 @@ class SubscriptionService(
-                    BadRequestDataException(invalidSubscriptionAttributeMessage(subscriptionId, it.key))
-                        .left().bind<Unit>()
+        }.plus("modifiedAt" to ngsiLdDateTime())
+            .forEach {
+                when {
+                    it.key == "geoQ" ->
+                        parseGeoQueryParameters(it.value as Map<String, String>, contexts).bind()
+                            ?.let { upsertGeometryQuery(it, subscriptionId).bind() }
+                    it.key == "notification" -> {
+                        val notification = it.value as Map<String, Any>
+                        updateNotification(subscriptionId, notification, contexts).bind()
+                    }
+                    it.key == "entities" -> {
+                        val entities = it.value as List<Map<String, Any>>
+                        updateEntities(subscriptionId, entities, contexts).bind()
+                    }
+                    it.key == "expiresAt" -> {
+                        val columnName = it.key.toSqlColumnName()
+                        val expiresAt = checkExpiresAtInTheFuture(it.value as String).bind()
+                        updateSubscriptionAttribute(subscriptionId, columnName, expiresAt).bind()
+                    }
+                    it.key == "watchedAttributes" -> {
+                        val value = (it.value as List<String>).map { watchedAttribute ->
+                            expandJsonLdTerm(watchedAttribute, contexts)
+                        }.toSqlValue(it.key)
+                        updateSubscriptionAttribute(subscriptionId, it.key.toSqlColumnName(), value).bind()
+                    }
+                    listOf(
+                        "subscriptionName",
+                        "description",
+                        "notificationTrigger",
+                        "timeInterval",
+                        "q",
+                        "scopeQ",
+                        "isActive",
+                        "modifiedAt",
+                        "throttling",
+                        "lang",
+                        "datasetId",
+                        "jsonldContext"
+                    ).contains(it.key) -> {
+                        val columnName = it.key.toSqlColumnName()
+                        val value = it.value.toSqlValue(it.key)
+                        updateSubscriptionAttribute(subscriptionId, columnName, value).bind()
+                    }
+                    listOf("csf", "temporalQ").contains(it.key) -> {
+                        NotImplementedException(unsupportedSubscriptionAttributeMessage(subscriptionId, it.key))
+                            .left().bind<Unit>()
+                    }
+                    else -> {
+                        BadRequestDataException(invalidSubscriptionAttributeMessage(subscriptionId, it.key))
+                            .left().bind<Unit>()
+                    }
                 }
             }
-        }
```

Comment:
yes, when I updated this (for the transformations of attributes in `CompactedEntity`) : https://github.com/stellio-hub/stellio-context-broker/pull/1261/files#diff-cbcd7396262b86fcf227a66470e0e8c10dc9e4ceb9d0ff85720323a7562e92cfL99-R117, it broke the update of the subscriptions because `modifiedAt` was no longer updated since it was filtered because it is part of JSONLD_COMPACTED_ENTITY_CORE_MEMBERS.

(btw, it only appears as a big change because of the change in indentation)